sdk/appconfiguration/app-configuration/package.json

```diff
@@ -91,12 +91,12 @@
     "@azure/core-http": "^2.0.0",
     "@azure/core-paging": "^1.1.1",
     "@azure/core-tracing": "1.0.0-preview.13",
+    "@azure/identity": "2.0.0-beta.5",
```

Comment:
Why is this depending on identity directly?

Reply:
![image](https://user-images.githubusercontent.com/10452642/126562640-e30b48a0-64c5-43e4-a444-c50e45d8ab97.png)

Reply:
I guess I can get that from core-auth.

Reply:
Yeah, we shouldn't be depending on identity except as a dev dependency. Looks like we probably got lucky that the transpiled code removed the TokenCredential import, but we should get it from core-auth.